pvlib/irradiance.py

```diff
@@ -662,7 +667,7 @@ def klucher(surface_tilt, surface_azimuth, dhi, ghi, solar_zenith,
     Returns
     -------
     diffuse : numeric
-        The sky diffuse component of the solar radiation. [Wm⁻²]
+        The sky diffuse component of the solar radiation [Wm⁻²].
```

Comment:
Where are these periods supposed to go? Before or after the units? I see a mix in the docs

Reply:
I vote for after.

Reply:
The recently added [documentation example](https://pvlib-python.readthedocs.io/en/latest/contributing/style_guide.html) featured the period before the units.

I think this probably deserves a vote, mostly just a final decision.

Reply:
> The recently added [documentation example](https://pvlib-python.readthedocs.io/en/latest/contributing/style_guide.html) featured the period before the units.
> 
> I think this probably deserves a vote, mostly just a final decision.

Ah I missed that.
Could react here?
👍 for after `solar radiation [Wm⁻²].` ...
👎 for before `solar radiation. [Wm⁻²]` ...
When the units are at the end of the sentence, ie no subsequent text, either way looks fine. When there's text after the units, I think period after is better, so for consistency I just thought `[Wm⁻²].` made sense. I don't feel too strongly either way though, if someone else does, I'm happy just to follow that 😅

Reply:
I can live with either.

Reply:
Units before the period suggests that it's a member of the preceding sentence and should obey its grammatical structure. Consider this example from `pvlib.temperature.sapm_cell`:

https://github.com/pvlib/pvlib-python/blob/b50b56f6cc30f11c39935c736854f22d5810c44a/pvlib/temperature.py#L72-L73


The unit could be confused as a parenthetical describing the thing before it (`a height of 10 meters` in this case), which of course is not the intended meaning.  

Fundamentally I think the unit describes the parameter, not something in the parameter description text, and we are only including the unit immediately adjacent to the description because sphinx doesn't offer a more structured alternative.  For example, in a different documentation system that represents parameters in tabular form, we might have chosen to have description and unit in separate columns.  Since we don't actually have tables, all we can do is to put them next to each other.  But I don't see why it makes sense to go a step further and include the units *inside* (inside of after) the description text.

Not sure that's convincing to others, but anyway I'm +1 to period before units.